packages/js/product-editor/src/blocks/catalog-visibility/index.ts

```diff
@@ -6,7 +6,7 @@ import { BlockConfiguration } from '@wordpress/blocks';
 /**
  * Internal dependencies
  */
-import { initBlock } from '../../utils/init-blocks';
+import { initBlock } from '../../utils/init-block';
```

Comment:
There are a lot of files where this import got changed.
We had duplicate `initBlock` functions one in the `init-blocks` file and another in the `init-block` file. I removed an older one in `init-block` but then renamed the `init-blocks` file to `init-block` to avoid confusion.